public/app/plugins/datasource/azuremonitor/azure_monitor/response_parser.ts

```diff
@@ -29,7 +29,7 @@ export default class ResponseParser {
     for (let i = 0; i < result.value.length; i++) {
       if (!find(list, ['value', get(result.value[i], valueFieldName)])) {
         const value = get(result.value[i], valueFieldName);
-        const text = get(result.value[i], textFieldName, value);
+        const text = get(result.value[i], textFieldName, value)?.toLowerCase();
```

Comment:
why does this need to be lowercase now?

Reply:
I assume this needs to be lower case as the resource graph query may return them in varying cases - @alyssabull can you confirm?